rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -1,16 +1,28 @@
-Name:		securedrop-workstation-dom0-config
-Version:	0.5.3
+%global srcname securedrop-workstation-dom0-config
+%global version 0.5.3
+%global __python3 /usr/bin/python3
+%global python3_sitelib /usr/lib/%{_python_version}/site-packages
+# For reproducible builds
```

Comment:
if there's a source or description for these changes in a comment or commit message, it may help future maintainers.

Reply:
Good point, I'll amend the commit message with specifics. And the `buildflags` actually isn't required anymore, I'd documented that based on compared builds between F25 & F32, but rpms are reproducible without that particular override.

Reply:
Added comments inside the specfile. Ended up preserving optflags, too.